docs/user-guide/ze-profiles.md

```diff
@@ -67,3 +86,27 @@ Activate the SCS plug-in in Zowe Explorer.
 6. Restart VSCode.
 
 The credentials of your newly created or existing profiles are now stored securely.
+
+## Use Base Profile and Token with Existing Profiles
+
+As a Zowe user, you can leverage the base profile functionality to access multiple Zowe services instantly. Base profiles enable you to authenticate via Zowe API Mediation Layer. You can use base profiles with more than one service profile. Any service profile with the same host and port as your base profile automatically uses your API ML token to log in.
+
+Before using the base profile functionality, ensure that you have [Zowe CLI](https://docs.zowe.org/stable/user-guide/cli-installcli.html) v6.16 or higher installed.
+
+Connect your service profile with a base profile and token.
```

Comment:
Should this be a heading? ### ?

Reply:
And maybe "Log in and connect your service profile..."

Reply:
Maybe you could say "How to connect..." and make it a heading, otherwise it is sort of confusing why this sentence is here P:

Reply:
I didn't want to entitle such a small paragraph, really. Does it rather confuse you?

Reply:
Opinion: Rename the existing heading to something catchy that a user wants to do, like "Access services through API ML with SSO" or similar.
And move that rogue sentence into the above paragraph so that it doesn't stick out. The paragraph explains the concept and then "in order to make this work, you need to connect your service profile with a base profile. "